app/src/main/java/ar/com/wolox/android/cookbook/room/database/RoomDataEntity.kt

```diff
@@ -0,0 +1,23 @@
+package ar.com.wolox.android.cookbook.room.database
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+/**
+ * Entity represents data for a single table row, constructed using an annotated java data object.
+ * Each entity is persisted into its own table.
+ */
+@Entity(tableName = "personal_notes")
+class RoomDataEntity {
```

Comment:
Name it on a better way. I understand it's an example, but let name it as you're using on the real life.